### PR TITLE
Added references to the recently released multioutput paper

### PR DESCRIPTION
**Pull request by markvdw**
Added some references to the recently released multioutput paper.

I also fixed some broken links in `intro.rst`.

Also, the maths in the multioutput notebook is broken in the docs, but it renders fine in the notebook. Does anyone have an idea why this could happen?

### CODE REVIEW

**Comment by st--** [comment]
> Also, the maths in the multioutput notebook is broken in the docs, but it renders fine in the notebook. Does anyone have an idea why this could happen?

Having looked at the source of the [nbsphinx documentation on markdown cells](https://github.com/spatialaudio/nbsphinx/blob/0.5.1/doc/markdown-cells.ipynb) (which is itself a jupyter notebook), I think this is because it requires a slightly different syntax for equations.

We are using e.g. `$$f(x) = W g(x),$$` for display equations, whereas it should be
```
\begin{equation}
f(x) = W g(x),
\end{equation}
```

Would you be up for helping us out by trying to fix this in the notebook(s)?

File: README.md [hunk 1]
```diff
@@ -141,3 +143,19 @@ To cite GPflow, please reference the [JMLR paper](http://www.jmlr.org/papers/vol
   url     = {http://jmlr.org/papers/v18/16-537.html}
 }
 ```
+
+Since the publication of the GPflow paper, the software has been significantly extended
+with the framework for interdomain approximations and multiouput priors. We review the
```
Comment (st--) [inline README.md]:
```suggestion
with the framework for interdomain approximations and multioutput priors. We review the
```

File: doc/source/intro.rst [hunk 2]
```diff
@@ -64,10 +64,20 @@ The following table summarizes the model options in GPflow.
...
 A unified view of many of the relevant references, along with some extensions, and an early discussion of GPflow itself, is given in the PhD thesis of `Matthews <http://mlg.eng.cam.ac.uk/matthews/thesis.pdf>`_ [8].
 
+Interdomain inference and multioutput GPs
+"""""""""""""""""""""""""""""""""""""""""
+GPflow has an extensive and flexible framework for specifying interdomain inducing variables for variational approximations.
+Interdomain variables can greatly improve the effectiveness of a variational approximation, and are used in e.g.
+`convolutional GPs <notebooks/advanced/convolutional.html>`_. In particular, they are crucial for defining sensible sparse
+approximations for `multioutput GPs <notebooks/advanced/multioutput.html>`_.
+
+GPflow has a unifying design for using multitoutput GPs and specifying interdomain approximations. A review of the
```
Comment (st--) [inline doc/source/intro.rst]:
```suggestion
GPflow has a unifying design for using multioutput GPs and specifying interdomain approximations. A review of the
```

File: doc/source/intro.rst [hunk 3]
```diff
@@ -64,10 +64,20 @@ The following table summarizes the model options in GPflow.
...
+mathematical background and the resulting software design is described in a paper on `arxiv <https://arxiv.org/abs/2003.01115>`_ [9].
```
Comment (st--) [inline doc/source/intro.rst]:
```suggestion
mathematical background and the resulting software design is described in a `paper on arXiv <https://arxiv.org/abs/2003.01115>`_ [9].
```

File: doc/source/intro.rst [hunk 4]
```diff
@@ -92,6 +102,24 @@ To cite GPflow, please reference the `JMLR paper <http://www.jmlr.org/papers/vol
         url     = {http://jmlr.org/papers/v18/16-537.html}
     }
 
+Since the publication of the GPflow paper, the software has been significantly extended
+with the framework for interdomain approximations and multiouput priors. We review the
```
Comment (st--) [inline doc/source/intro.rst]:
```suggestion
with the framework for interdomain approximations and multioutput priors. We review the
```

File: doc/source/notebooks/advanced/multioutput.pct.py [hunk 5]
```diff
@@ -18,7 +18,31 @@
...
+# - extensible, by allowing new interdomain variables and kernels to be specified while reusing exising code where possible,
+# - efficient, by allowing the most efficient custom code path to be specified where desired.
+#
+# Getting to grips with the maths and code can be a bit daunting, so to accompany the documentation there is an in-depth review on [arXiv](https://arxiv.org/abs/2003.01115), which provides a unified mathematical framework, together with a high-level description of software design choices in GPflow.
+#
+# This notebook shows the various design choices that can be made, to show the reader the flexibility of the framework. This is done in the hope that an example is provided that can be easily adapted to the special case that the reader wants to implement.
+#
+# A reader who just wants to use a multioutput kernel should simply choose the most efficient set of inducing variables.
+#
+# To cite this framework, please reference our [arXiv](https://arxiv.org/abs/2003.01115) paper.
```
Comment (st--) [inline doc/source/notebooks/advanced/multioutput.pct.py]:
```suggestion
# To cite this framework, please reference our [arXiv paper](https://arxiv.org/abs/2003.01115).
```

File: doc/source/notebooks/advanced/multioutput.pct.py [hunk 6]
```diff
@@ -18,7 +18,31 @@
...
+# This notebook shows how to construct a multi-output GP model using GPflow, together with different interdomain inducing variables which lead to different approximation properties. GPflow provides a framework for specifying multioutput GP priors, and interdomain approximations which is
+# - modular, by providing a consistent interface for the user of the resulting `SVGP` model,
+# - extensible, by allowing new interdomain variables and kernels to be specified while reusing exising code where possible,
+# - efficient, by allowing the most efficient custom code path to be specified where desired.
+#
+# Getting to grips with the maths and code can be a bit daunting, so to accompany the documentation there is an in-depth review on [arXiv](https://arxiv.org/abs/2003.01115), which provides a unified mathematical framework, together with a high-level description of software design choices in GPflow.
```
Comment (st--) [inline doc/source/notebooks/advanced/multioutput.pct.py]:
```suggestion
# Getting to grips with the maths and code can be a bit daunting, so to accompany the documentation there is an [in-depth review on arXiv](https://arxiv.org/abs/2003.01115), which provides a unified mathematical framework, together with a high-level description of software design choices in GPflow.
```

**Comment by markvdw** [comment]
Great, I'll take a look at fixing the equations in the multioutput notebook as well.

**Comment by markvdw** [comment]
@st-- when you're happy, can you press the button?

**Pull request merged**